getdata_HN.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import json
+import os.path
+from urllib.parse import urljoin
+
+URL = 'https://hacker-news.firebaseio.com/v0/'
+KEYWORDS = ['python', 'django']
+story = {}
+
+
+def get_json(param):
+    url = urljoin(URL, param)
+    req = requests.get(url)
+    return req.json()
+
+
+def get_stories():
+    parameter = 'topstories.json?print=pretty'
+    return get_json(parameter)
+
+
+def check_item():
+    for item in get_stories():
+        parameter = 'item/{}.json?print=pretty'.format(item)
+        item = get_json(parameter)
+        for kwd in KEYWORDS:
+            if kwd in item['title']:
+                return item
+
+
+def get_item(item):
+    parameter = 'item/{}.json?print=pretty'.format(item)
+    item = get_json(parameter)
+    story[item['id']] = {'title': item['title']}
+    return
+
+
+def save_items(items, filename='tmp-data.json'):
+    with open(filename, 'w') as outfile:
+        json.dump(items, outfile, indent=4)
+    return
+
+
+if os.path.isfile('tmp-data.json'):
```

Comment:
you should add this inside a `def main` and call main in 
`if __name__ == "__main__":`